packages/sources/por-address-list/schemas/env.json

```diff
@@ -8,6 +8,12 @@
     "RPC_URL": {
       "type": "string",
       "description": "The RPC URL to connect to the EVM chain the address manager contract is deployed to."
+    },
+    "CHAIN_ID": {
+      "required": false,
+      "description": "The chain id to connect to",
+      "type": "string",
+      "default": "1"
```

Comment:
Is this reflected in this EA's README?

Reply:
Does it get auto-generated after the PR is merged?

Reply:
Yes, i didn't change readmes where it gets generated automatically. According to [docs](https://github.com/smartcontractkit/external-adapters-js/tree/develop/packages/scripts#auto-generation-1) it should do it after merge to develop

Reply:
You should be able to see the changes once the changeset PR is updated after this is merged